mobileversion.html

```diff
@@ -0,0 +1,186 @@
+             <p class="mid">&#8226;</p>
+             <p class="back">Back End Dev</p>
+             <p class="mid">&#8226;</p>
+             <p class="year">2015</p>
+             </p>
+          </section>
+          <section class="day">
+             <p class="daily">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+             </p>
+          </section>
+          <section class="maincon">
+             <ul>
+          <li class="container2"><a href="#">html</a></li>
+          <li class="container3"><a href="#">css</a></li>
+          <li class="container4"><a href="#">javaScript</a></li>
+          </ul>
+       </section>
+       <section class="see">
+          <a href="#" class="pro">See Project</a>
+       </section>
+       </div>
+       </div>
+       <div class="container">
+        <div class="item1">
+            <img src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAkGBwgHBgkIBwgKCgkLDRYPDQwMDRsUFRAWIB0iIiAdHx8kKDQsJCYxJx8fLT0tMTU3Ojo6Iys/RD84QzQ5Ojf/2wBDAQoKCg0MDRoPDxo3JR8lNzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzf/wAARCAApAEADASIAAhEBAxEB/8QAHAAAAwEAAgMAAAAAAAAAAAAAAAUHBgEDAgQI/8QAMBAAAQMDAgQEBgEFAAAAAAAAAQIDBAAFERIhBiIxQRNRYYEHFEJxkaEjFbHB0fD/xAAYAQEBAQEBAAAAAAAAAAAAAAAFAwIEAf/EACsRAAEDAgQFAgcAAAAAAAAAAAEAAgMEEQUhMUESE1Fh8AZxFCIjgZHR8f/aAAwDAQACEQMRAD8AuJIAydhXW801JZU0+2h1pY5kLSFJUPUHrWAul7mXIlLi/DZPRpHT3860XCNy+ZiGG6f5WBy57o7fjp+KyH55JCbDpIoeY4+4TFNjtCVBSbVBBByCIyNj+KYUUqvvENssLHiXGQEKI5Gk8y1/ZP8AnpVgJJXBouT+UdkE1rhKkqGUkEZIyD3HWozxL8QrndtbEDMCIdsIV/Ksequ32H5NMfhPxB8tKXZJKsNPkrjk/Svun3G/3B86SfhE0cBldqNuymJQXWVWoooolVXz+visyFkwkJQ1nl8QZUR64OB/29aywz5LTMK56QlaskhPRQBII9wKTM8A3F20sQoFpSwt0oUZskBKmk7ElX1FRGRoGw3zjAzsDaUW+3MwUqKwy2EBaupIHX36+9JV76QwNbCyzgfvYfvumKaWUzkSv4gRn0z82Xbx/wAT3O2WyHIs6W0xpYIMojUpCsZCQDsD13Oeh2qRyH3pL635Lq3XlnKnHFFSlH1Jqp2RTF6tkmxT+VmSCWVHcoWN8j3GfY+dTGZBkQrg7AlJ0PtOeGsds+Y9O9K4HLE6ItAs4a9wh8RpnU8xYdNkR4EmUnMVpb2ASrSNk/c9K8VtS4D7bikuMOoUFIX0wRuCDVHtbEduGluKnS0tJSE56K04/vv7571luKnk/LsRkjLrjmsDuAAR+yf0aMoPUdRVYiaQxWaCRnqANzt5qVZ2HxspOdx3OXtnsqzwlfW+ILIzMBSHhySED6XB19j1Hoa7LhfY8bKGMPOjyPKPuf8AVYD4WwbvHmOvrj6bVJb0u+McazvpKR364z0wae3WL/TpTiFnDQ5kqUdtNAepZJKF5FMQQTrrbsqUETJT9T+rXNKWhzSpJKTtnFI+IEO6FqZZcUcHASgnJrTUV1EXWYp+W/isphGgzY4b8ONJC28EKDSsgjv0rnj+yv3aBGv8OI8JiAGpTKWzqO+ygMZOD+iPKqdRV6Sd9NKJGqtbV/FtAc2x6qQ2l+6NQ3nm7JPW+gJJa8BYCjkbjKfLNe5bOF590uaZM2CqGw0dOt0cy0gkgJB3HXc4FVKitvqBzXTRts85X1Nunma4iSQG3+UbDQnql7EcMN6EJV6k96S8d8Pu8RWEtRtQmR1a2k6sBzzSc7dP2BWqorhbEAvZTzWlrt1//9k=" alt="multiimage">
+        <section class="ton">
+            <h2 class="mps">Multi-Post<br>
+                Stories</h2>
+        </section>
+        <section class="sec">
+           <p class="mainp">
+           <h5 class="canopy">CANOPY</h5>
+           <p class="mid">&#8226;</p>
+           <p class="back">Back End Dev</p>
+           <p class="mid">&#8226;</p>
+           <p class="year">2015</p>
+           </p>
+        </section>
```

Comment:
- [ OPTIONAL] **You can use here ul tag with li tag rather than a section tag**
```
<ul class="sec">
            <li class="canopy">CANOPY</li>
            <li class="mid">&#8226;</li>
            <li class="back">Back End Dev</li>
            <li class="mid">&#8226;</li>
            <li class="year">2015</li>
</ul>
```